src/components/safe-apps/AddCustomAppModal.tsx

```diff
@@ -73,7 +75,7 @@ const AddCustomAppModal = ({ open, onClose, onSave, safeAppsList }: Props) => {
     })
   }, [chainId, debouncedUrl])
 
-  const appLogoUrl = safeApp?.iconUrl || APP_LOGO_FALLBACK_IMAGE
+  const appLogoUrl = safeApp?.iconUrl || (isDarkMode ? APP_LOGO_FALLBACK_DARK_IMAGE : APP_LOGO_FALLBACK_IMAGE)
```

Comment:
Please use the same image for both modes. You need to add special classes from global.css to the SVG elements. See how we did it for all other icons.

Reply:
Was using the img fallback component but changing to the SvgIcon seems to work just fine. done , thanks